app/Http/Controllers/StaticSitesController.php

```diff
@@ -21,4 +21,9 @@ public function showImpressum(): Response
     {
         return Inertia::render('Impressum');
     }
+
+    public function showProfessionalFieldDecision(): Response
```

Comment:
Function is not used. The related inertia site does not look finish. Why is it included in the PR?